### ISSUE TITLE
"Show duration" kills buttons in audio and video demos

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#17]
```
Please describe the issue:

I'm running Chrome 5.0.375.70 on Vista64.
If I go to http://playground.html5rocks.com/#video_tag or 
http://playground.html5rocks.com/#audio_tag, click "Play", wait for it to start 
playing, then click "Show duration", playback stops, the video disappears, and 
none of the 3 lower buttons ever works again.  However, clicking the play 
/icon/ still works.

Please provide any additional information below.

```

Original issue reported on code.google.com by `ericu@google.com` on 23 Jun 2010 at 12:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `erne...@google.com` on 5 Aug 2010 at 10:45
- Added labels: **SampleCode**

**Comment by GoogleCodeExporter** [comment]
Original comment by `erne...@google.com` on 17 Aug 2010 at 8:46
- Added labels: **Milestone-3**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 8 Sep 2010 at 6:12

**Comment by GoogleCodeExporter** [comment]
```
Removed document.body.innerHTML += on audio and video playground samples.
Fixed in change 059f9b9ba3

```

Original comment by `erne...@google.com` on 13 Sep 2010 at 7:01
- Changed state: **Fixed**
